### PR TITLE
[PVR] Some context menu cleanup

### PR DESCRIPTION
**Pull request by ksooo**
1st commit
- as discussed at DevCon, "Add timer" everywhere was renamed to "Record", "Add custom timer" to just "Add timer". Functionality behind is unchanged. "Record will schedule a one-time epg-based timer that will fire instantly or later, dependent on the start time of the selected epg-event. "Add timer" will open the timer settings dialog with a recurring epg-based timer preselected for creation.

screenshot004

2nd commit
- cleanup record/stop recording to use always the same string resources, eliminating duplicates we had so far.

@da-anda, @kib  good to go?

### CODE REVIEW

File: addons/resource.language.en_gb/resources/strings.po [hunk 1]
```diff
@@ -1164,6 +1164,11 @@ msgctxt "#263"
 msgid "Allow remote control via HTTP"
 msgstr ""
 
+#. Label of a context menu entry to start/schedule a recording
+#: xbmc/music/windows/GUIWindowMusicBase.cpp
+#: xbmc/pvr/dialogs/GUIDialogPVRGuideInfo.cpp
+#: xbmc/pvr/windows/GUIWindowPVRGuide.cpp
+#: xbmc/pvr/windows/GUIWindowPVRSearch.cpp
 msgctxt "#264"
```
Comment (da-anda) [inline addons/resource.language.en_gb/resources/strings.po]: have you checked if this label is used anywhere else as the places you listed? As it's a rather old label I suspect there might be other usecases

**Comment by da-anda** [comment]
thanks, looks good.

Because I just noticed in your screenshot - does it make sense to add an EPG entry to your favourites?

File: addons/resource.language.en_gb/resources/strings.po (hunk 1 above)
Comment (tamland) [inline addons/resource.language.en_gb/resources/strings.po]: on that note can we *please* only use ids with 5 or more digits? it's so hard to search for these numbers

Comment (ksooo) [inline addons/resource.language.en_gb/resources/strings.po]: > have you checked if this label is used anywhere else as the places you listed

Yes, I did. No other occurences found.

Comment (ksooo) [inline addons/resource.language.en_gb/resources/strings.po]: > on that note can we please only use ids with 5 or more digits

 Makes sense in general, but not for this particular case I have not introduced this id in this PR, just added documentation.

**Comment by ksooo** [comment]
> does it make sense to add an EPG entry to your favourites

Imo not. Also stubled over it last night, but then forgot.

@Montellese this menu item slips in from here https://github.com/xbmc/xbmc/blob/master/xbmc/windows/GUIMediaWindow.cpp#L1553 would it be okay to filter pvr windows out here, like already done for other "protocols"?

**Comment by Montellese** [comment]
Sure.

**Comment by ksooo** [comment]
@da-anda my take on "Add to favourites" in PVR wondows context menus:

* senseful: channels, recordings
* not senseful: epg entries, timers

Your take on this?

**Comment by da-anda** [comment]
@ksooo sounds good to me. Not to forget the folder of grouped recordings (like when grouped by name of show)

**Comment by Jalle19** [comment]
I agree with everything said so far, including not being able to add EPG tags to favourites.

**Comment by zag2me** [comment]
Just wanted to say this is a massive improvement in usability!

**Comment by jjd-uk** [comment]
As I mentioned in your presentation at Devcon, I found the old "Add timer" and "Add custom timer" odd, so this looks like a big improvement to me also.

**Comment by ksooo** [comment]
Timers and EPG events now no longer have  "Add to favourites".

**Comment by ksooo** [comment]
jenkins build this please

**Comment by kib** [comment]
@ksooo this is great, exactly what i meant.

**Comment by da-anda** [comment]
feel free to merge if jenkins is happy - don't think there are any objections from a usability POV

**Comment by MartijnKaijser** [comment]
looks ok

**Pull request merged**